UnityProject/Assets/Scripts/Objects/Atmospherics/AirController.cs

```diff
@@ -263,7 +263,8 @@ public bool WillInteract(HandApply interaction, NetworkSide side)
 
 		public void ServerPerformInteraction(HandApply interaction)
 		{
-			if (accessRestrictions.CheckAccessCard(interaction.HandObject))
+			if (accessRestrictions.CheckAccessCard(interaction.HandObject) ||
```

Comment:
I meant here, to check for AI before the HandObject but honestly it doesn't really matter. Ignore the comment.

Reply:
Looks like all this will do is toggle the lock on the ACU, but they still can't open the UI. I think the fix will unfortunately be a little more involved.

Reply:
What is this this? 🤔 

![image](https://user-images.githubusercontent.com/34368774/153625266-d65d5178-6925-47b9-bce1-6b5cba239b3c.png)